src/jdk.compiler/share/classes/com/sun/tools/javac/util/Name.java

```diff
@@ -38,7 +38,7 @@
  *  This code and its internal interfaces are subject to change or
  *  deletion without notice.</b>
  */
-public abstract class Name implements javax.lang.model.element.Name, PoolConstant {
+public abstract class Name implements javax.lang.model.element.Name, PoolConstant, Comparable<Name> {
```

Comment:
It's disappointing that we can't easily make `javax.lang.model.element.Name` implement `Comparable`